geomstats/backend/numpy_random.py

```diff
@@ -1,19 +1,3 @@
 """Numpy based random backend."""
 
-import numpy as np
-
-
-def rand(*args, **kwargs):
-    return np.random.rand(*args, **kwargs)
-
-
-def randint(*args, **kwargs):
-    return np.random.randint(*args, **kwargs)
-
-
-def seed(*args, **kwargs):
-    return np.random.seed(*args, **kwargs)
-
-
-def normal(*args, **kwargs):
-    return np.random.normal(*args, **kwargs)
+from numpy.random import normal, rand, randint, randn, seed  # NOQA
```

Comment:
If we do this change, shouldn't we do it in all the backends, otherwise we introduce style inconsistency. What do you think?

Reply:
Jepp, see https://github.com/geomstats/geomstats/pull/326.